services/inference/Makefile

```diff
@@ -62,6 +62,8 @@ install_dev: boilerplate
 clone_models:
 	git lfs install || true
 	git clone https://huggingface.co/ibm/test-tsfm mytest-tsfm || true
+	git clone https://huggingface.co/amazon/chronos-t5-small mytest-tsfm/chronos-t5-small || true
```

Comment:
Instead, can we update `test-tsfm` to have a `chronos-t5-small` folder containing the tsfm-config already?

Reply:
@wgifford I am trying to push tsfm_config.json to test_tsfm/chronos-t5-small/tsfm_config.json and create a PR. Could you give me access to this repository?

Having duplicated copy of chronos model in this repository may not be ideal. I don't see a way to use submodules given we need to add tsfm_config.json as well. 

We can handle this in makefile: 
1. Persist tsfm_config files for all models in a separate directory: tsfm_config_files/chronos/tsfm_config.json.
2. Clone chronos model repository -> we can do submodules too.
3. Copy tsfm_config_files/chronos/tsfm_config.json to chronos model path.

What do you think?

Reply:
We moved this file to test-tsfm repository with this content:
```
{
    "service_handler_module_path": "tsfminference.chronos_service_handler",
    "service_handler_class_name": "ChronosForecastingHandler",
    "minimum_context_length": 512,
    "module_path": "chronos",
    "model_type": "chronos-t5-tiny",
    "model_config_name": "T5Config",
    "model_class_name": "ChronosPipeline"
}
```

Thank you @wgifford